store/disk.go

```diff
@@ -109,3 +110,22 @@ func (c *onDiskStore) Update(oldID, newID string) error {
 		filepath.Join(c.path, hashString(newID)),
 	)
 }
+
+func (c *onDiskStore) Delete(ids ...string) error {
+	if c.sem != nil {
+		c.sem.Lock()
+		defer c.sem.Unlock()
+	}
+
+	for _, id := range ids {
+		if err := os.Remove(filepath.Join(c.path, hashString(id))); err != nil {
```

Comment:
`os.RemoveAll` is cleaner here as it returns `nil` if the path doesn't exist (avoids explicitly handling that).

Reply:
Thanks for pointing that out. Fixed.